packages/core/components/FileMetadataSearchBar/test/FileMetadataSearchBar.test.tsx

```diff
@@ -116,26 +113,30 @@ describe("<FileMetadataSearchBar />", () => {
         expect(getByText("Sun Aug 30 2020")).to.not.be.empty;
     });
 
-    it("defaults end date to start date when only start date is chosen", async () => {
+    it("creates RANGE() file filter of RANGE(day,day+1) when only start date is selected", async () => {
         // Arrange
         const { actions, logicMiddleware, store } = configureMockStore({ state: initialState });
         const { getByText } = render(
             <Provider store={store}>
                 <FileMetadataSearchBar />
             </Provider>
         );
-        const day = 17;
-        const date = new Date();
-        const expectedDate = `${date.getFullYear()}-${("0" + (date.getMonth() + 1)).slice(
-            -2
-        )}-${day}`;
-        const expectedRange = `${expectedDate}${DATE_RANGE_SEPARATOR}${expectedDate}`;
+        const startDate = new Date();
+        const day = 15; // Arbitrary day of the month that isn't likely to render twice
+        startDate.setDate(day);
+        startDate.setHours(0);
+        startDate.setMinutes(0);
+        startDate.setSeconds(0);
+        startDate.setMilliseconds(0);
+        const endDate = new Date(startDate);
+        endDate.setDate(endDate.getDate() + 1);
```

Comment:
so endDate is one day after start day, but in the old test, it was one MONTH later?  Just making sure I understand.

Reply:
I don't honestly know why it did that. The expectation in this test is (and was) for the start date and the end date to be the same in the UI if you only have a start date selected